wemake_python_styleguide/logic/naming/logical.py

```diff
@@ -109,7 +109,15 @@ def is_too_short_name(
     >>> is_too_short_name('xy', min_length=2, trim=True)
     False
 
+    >>> is_too_short_name('np', min_length=3)
+    False
+
     """
+    allow_list = ['np', 'pd', 'df', 'plt', 'sns', 'tf', 'cv']
```

Comment:
We tend to move these values to https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/constants.py

Reply:
ok will move them

Reply:
@sobolevn You can check the changes now